client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -643,4 +643,17 @@ private void applyDefaultExpirationFlags(long lifespan, long maxIdle) {
        }
        return Collections.unmodifiableSet(toReturn);
    }
+
+	@Override
+	public <T> T execute(String taskName, Map<String, ?> params) {
+		assertRemoteCacheManagerIsStarted();
+		Map<String, byte[]> marshalledParams = new HashMap<>();
+		if (params != null) {
+   		for(java.util.Map.Entry<String, ?> entry : params.entrySet()) {
```

Comment:
Seems you have tabs instead of whitespace ?  Also this block should be indented 1 more time.